docs/source/writing_benchmarks.rst

```diff
@@ -244,3 +246,69 @@ garbage collector at a given state::
 
 - ``unit``: The unit of the values returned by the benchmark.  Used
   for display in the web interface.
+
+Multiple
+````````
+
+There is also a special benchmark type for benchmarks that you want to
+test in multiple ways, for example to test both for memory usage and
+run time.  "Multiple" benchmarks use the prefix ``multi`` and have a
+required attribute ``types``, which is a list of benchmark types to
+run.
```

Comment:
`type` is a Python keyword, so maybe rename the `types` attribute to something else like `multi` or `measurements`?
(not sure ... if you like `types` better it's totally fine)